apps/frontend/src/components/SendForm.svelte

```diff
@@ -1,27 +1,36 @@
 <script lang="ts">
   import { getContext } from 'svelte';
   import { Chain, CreateTxReturn } from '@baf-wallet/interfaces';
-  import Button from './base/Button.svelte';
+  import Button from '@baf-wallet/base-components/Button.svelte';
+  import Lazy from '@baf-wallet/base-components/Lazy.svelte';
   import TxModal from './TxModal.svelte';
-  import SendNearFormPart from './near/SendNearFormPart.svelte';
+  import { ChainStores } from '../state/chains.svelte';
+  import { SiteKeyStore } from '../state/keys.svelte';
 
-  let createTX: <T>() => Promise<CreateTxReturn<T>>; 
+  let createTX: <T>() => Promise<CreateTxReturn<T>>;
   export let postSubmitHook: () => void | undefined;
   export let onCancel: () => void | undefined;
   export let chain: Chain;
 
+  let chainSendFormPart;
   const { open } = getContext('modal');
 
-  const handleSubmit = async (v: any) => {
-    console.log(`submit: ${v}`);
+  // TODO: clean up imports, see https://github.com/bafnetwork/baf-wallet-v2/issues/54
+  const ChainSendFormPart = (chain: Chain) => () =>
```

Comment:
oof yeah I feel like we definitely want a better system for this, but I think it's ok for now - we have an issue and in the long run I think this is somewhat OK if we're careful / clean about not letting this jank leak elsewhere.

Reply:
Yeah agreed